retailer-oas/retailer.yaml

```diff
@@ -376,58 +519,170 @@ paths:
+                      - "MAX_ORDERS_EXCEEDED"
+                      - "INVALID_PRODUCT_QUANTITY"
+                      - "ORDER_TOTAL_MISMATCH"
+                      - "DUPLICATE_ORDER"
+                  errors:
+                    type: array
+                    items:
+                      type: string
+                      description: List of error details
 
 components:
-  schemas:      
+  schemas:
+
     AccessToken:
       type: object
       properties:
-        token: 
+        token:
           type: string
-        tokenType: 
+        tokenType:
           type: string
-        scope: 
+        scope:
           type: string
-        expiresIn: 
+        expiresIn:
           type: integer
-        createdAt: 
+        createdAt:
           type: integer
 
     PickingLocation:
       type: object
       properties:
         id:
-          type: string
+          type: integer
+          readOnly: true
+          description: The unique identifier of the picking location
         name:
           type: string
+          description: The name of the picking location.
+          example: "Warehouse 1"
         coords:
           $ref: '#/components/schemas/GeoCoordinates'
         postalCode:
           type: string
+          description: The postal code of the picking location
+          example: "90210"
         addressLine1:
           type: string
+          description: The first line of the address of the picking location
+          example: "123 Main St"
         addressLine2:
           type: string
-        serviceArea:
-          $ref: '#/components/schemas/ServiceArea'
+          description: The second line of the address of the picking location
+          example: "Suite 456"
+        countryCode:
+          type: string
+          format: ISO 3166-1
+          description: The country code of the picking location
+          example: "PL"
+        person_of_contact:
+          type: string
+          description: The person of contact at the picking location
+          example: "John Smith"
+        email:
+          type: string
+          format: email
+          description: The email of the person of contact
+          example: "john.smith@example.com"
+        phone:
+          type: string
+          format: phone
+          description: The phone number of the person of contact
+          example: "+1 555 555 5555"
+        has_parking:
+          type: boolean
+          description: Indicates if the picking location has parking available
+          example: true
+        status:
+          type: string
+          enum: [active, temporary_closed]
+          description: The status of the picking location, whether it is active or temporarily closed
+          example: "active"
+
+
+    PickingLocationSchedule:
+      type: object
+      properties:
+        id:
+          type: integer
+          description: The unique identifier of the picking location
+          example: 1
+        opening_hours:
```

Comment:
Should we adapt to scenarios where PLs might be closed during parts of the day? That's not the case for supermarkets usually but if we have a dark store that closes for lunch, we need more than a single set of opening/closing hours for the day.

Reply:
[Discussion result of Engineering + Product]

We need to iterate on this to adapt to this scenario.